volatility3/framework/interfaces/objects.py

```diff
@@ -115,7 +115,16 @@ def __init__(self, context: 'interfaces.context.ContextInterface', type_name: st
         mask = context.layers[object_info.layer_name].address_mask
         normalized_offset = object_info.offset & mask
 
-        self._vol = collections.ChainMap({}, {'type_name': type_name, 'offset': normalized_offset}, object_info, kwargs)
+        vol_info_dict = {'type_name': type_name, 'offset': normalized_offset}
```

Comment:
This is a lot of effort because you feel `get_symbol_table_name` is too long.  You have to type it once, but it'll get read far more often than that, so I'm all in favour of long names.  Also IDEs tend to autocomplete, so monkeying with things for the sake of saving a few letters isn't a good enough reason.  You're right though, we should probably make use of that method more (for example in cast).  If you really don't like seeing the split, I might consider a method for getting just the symbol name, but it'll be calculated in the method, not stored as additional data that could get out of sync.

Reply:
Removed the changes here